package.json

```diff
@@ -28,6 +28,7 @@
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
     "react-redux": "^7.0.1",
+    "react-share": "^4.1.0",
```

Comment:
this adds 18kb to the bundle, any way we could just include the twitter svg manually? https://bundlephobia.com/result?p=react-share@4.1.0